features/works/work_view.feature

```diff
@@ -5,6 +5,7 @@ Feature: View a work with various options
   Scenario: viewing a work in explicit View Full Work mode, with JavaScript turned off (Issue 2205)
   Given the chaptered work with 2 comments "Whatever"
   When I view the work "Whatever" in full mode
+  Then I should not see "Reindex Work"
```

Comment:
This definitely needs to go into its own scenario, or perhaps two. (The wording for the steps here isn't at all accurate.)

```
Scenario: Regular logged-in user doesn't have the option to reindex a work
Given I am logged in
  And I have the work "Whatever"
When I view the work "Whatever"
Then I should not see "Reindex Work"

Scenario: Logged-out user doesn't have the option to reindex a work
Given the work "Whatever"
  And I am logged out
When I view the work "Whatever"
Then I should not see "Reindex Work"
```